src/layout/Layout.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { Box } from '@chakra-ui/react'
+import Navigation from './Navigation'
+import Footer from './Footer'
+
+type LayoutProps = {
+  children: JSX.Element
+}
```

Comment:
JSX.Element has strict constraints that accept only one single JSX element.

You may need to use other props in the future or in the other components. You can refer to this [article](https://dev.to/debs_obrien/typescript-and-react-children-4epg).